test/cluster.go

```diff
@@ -10,7 +10,7 @@ import (
 	"github.com/fabric8-services/fabric8-cluster/configuration"
 	"github.com/fabric8-services/fabric8-common/httpsupport"
 	"github.com/jinzhu/gorm"
-	"github.com/satori/go.uuid"
+	uuid "github.com/satori/go.uuid"
```

Comment:
redundant

Reply:
same response as before: this is auto-added by the go tools from VSCode. Please let me know if that's an issue for you.